source/develop/release-management/features/metrics/metrics-store-installation.html.md

```diff
@@ -122,63 +84,101 @@ This procedure will define the curator pod so that it deletes metrics indexes th
 
-2. Install / Upgrade and activate one or more hosts, 4.1.3 and above.
+2. Requires installing / upgrading and activating one or more hosts, 4.1.8 and above.
+   Until 4.1.8 is available, You can use the snapshots repository to get the latest ovirt-engine-metrics package.
+   http://resources.ovirt.org/pub/ovirt-4.1-snapshot/rpm/
 
-3. Copy the CA certificate - created earlier on in this procedure [(see oVirt Metrics Store Setup)](https://github.com/ViaQ/Main/blob/master/README-install.md#getting-the-shared_key-and-ca-cert) - to the engine machine.
+3. Copy  /etc/ovirt-engine-metrics/config.yml.example  to config.yml.
 
+   On the oVirt engine machine, run:
 
-   On the metrics store machine, run:
+        # cp /etc/ovirt-engine-metrics/config.yml.example /etc/ovirt-engine-metrics/config.yml
 
-        # scp /path/to/ca/certificate.cert root@<fqdn/of/engine/machine>:/etc/ovirt-engine-metrics
+4. Update the file /etc/ovirt-engine-metrics/config.yml located on the engine machine, with the following lines:
 
-4. Copy  /etc/ovirt-engine-metrics/config.yml.example  to config.yml.
+        # vi /etc/ovirt-engine-metrics/config.yml
 
-   On the oVirt engine machine, run:
+5. Update the values with your specific environment details:
+
+
+- `ovirt_env_name:` **Use the same name you configured earlier**.
+
+  Environment name. Can be used to identify data collected in a single central
+  store sent from more than one oVirt engine.
+
+- `fluentd_elasticsearch_host:` (required - no default value)
+
+  Address or hostname (FQDN) of the Elasticsearch server host.
+
+- `ovirt_env_uuid_metrics:` (required - no default value)
+
+  UUID of the project/namespace used to store metrics records.
+  This is used to construct the index name in Elasticsearch.
+  For example, if you have ovirt_env_name: myenvname,
+  then in logging OpenShift you will have a project named ovirt-metrics-myenvname.
+  You need to get the UUID of this project like this:
+  oc get project ovirt-metrics-myenvname -o jsonpath='{.metadata.uid}'
 
-        # cp /etc/ovirt-engine-metrics/config.yml.example /etc/ovirt-engine/config.yml
+- `ovirt_env_uuid_logs:` (required - no default value)
 
-5. Update the file /etc/ovirt-engine-metrics/config.yml located on the engine machine, with the following lines:
+  UUID of the project/namespace used to store log records.
+  This is used to construct the index name in Elasticsearch.
+  For example, if you have ovirt_env_name: myenvname,
+  then in logging OpenShift you will have a project named ovirt-logs-myenvname.
+  You need to get the UUID of this project like this:
+  oc get project ovirt-logs-myenvname -o jsonpath='{.metadata.uid}'
 
-        # vi /etc/ovirt-engine-metrics/config.yml.example
+- `fluentd_elasticsearch_ca_cert_path:` (required - no default value)
 
-    Replace the example values with your specific environment details:
+  The path to the file containing the CA certificate of the CA that issued
```

Comment:
Not sure about this one.

Reply:
They need to add the patch to the file on the local machine.